src/hera/workflows/user_container.py

```diff
@@ -23,6 +23,31 @@ class UserContainer(_ModelUserContainer):
     resources: Optional[Union[Resources, ResourceRequirements]] = None  # type: ignore[assignment]
     volumes: Optional[List[_BaseVolume]] = None
 
+    def _build_image_pull_policy(self) -> Optional[str]:
```

Comment:
There's another one of these. This is not shared because it returns a `str` whereas the other returns an `ImagePullPolicy`. The type difference comes from upstream. Maybe we should edit the OpenAPI spec on this one too and only use strings?